ansible/roles/products/common/docker-container/tasks/create-container/no-setup-no-terminator.yml

```diff
@@ -8,6 +8,11 @@
   set_fact:
     shared_volumes: "{{ common_shared_volumes + container_type_shared_volumes + custom_shared_volumes }}"
 
+- name: Find out if we should record all data from these hands
+  set_fact:
+    record_hand_data: "enabled"
+  when: config_branch in bag_all_data_from
```

Comment:
cant we just bag data from hand that is in config branch is record_hand_data = enabled? Then we dont need to keep "bag_all_data_from" variable and remember to add branch names there etc.

Reply:
we can open a ticket to improve this